packages/graphql-server/src/errors/errors.ts

```diff
@@ -10,38 +11,44 @@ export const errorMessages = {
 }
 
 export const generateAuthenticationError = (traceId?: string) => {
-  const error = new AuthenticationError(`${traceId || ''} - ${errorMessages.notAuthorized}`)
+  const error = new AuthenticationError(`${errorMessages.notAuthorized}`)
   logger.info('generateAuthenticationError', { traceId, error: JSON.stringify(error) })
+  handleGraphQlError({ error: JSON.stringify(error), traceId, caller: 'generateAuthenticationError' })
   return error
 }
 
 export const generateUserInputError = (traceId?: string) => {
-  const error = new UserInputError(`${traceId || ''} - ${errorMessages.badRequest}`)
-  logger.info('generateAuthenticationError', { traceId, error: JSON.stringify(error) })
+  const error = new UserInputError(`${errorMessages.badRequest}`)
+  logger.info('generateUserInputError', { traceId, error: JSON.stringify(error) })
+  handleGraphQlError({ error: JSON.stringify(error), traceId, caller: 'generateUserInputError' })
   return error
 }
 
 export const generateValidationError = (traceId?: string) => {
-  const error = new ValidationError(`${traceId || ''} - ${errorMessages.badRequest}`)
+  const error = new ValidationError(`${errorMessages.badRequest}`)
```

Comment:
This may need some thoughts. This is a new error message object:
![image](https://user-images.githubusercontent.com/36730355/89539771-60a4ee80-d826-11ea-8c2d-b9e853bd366f.png)

This was the old one
![image](https://user-images.githubusercontent.com/36730355/89539908-8c27d900-d826-11ea-80e2-c8844254c435.png)

You will need to return `traceId`. It's important incase something broken when FE fetching from BE, and you want to trace which request lead to the error. The logger in AWS are very messy, `traceId` is there to simplify searching tasks

Reply:
Thank @phmngocnghia for review, but this ticket ask we show the friendly errors for the user. That why I remove the traceId from response error. 
FYI, when we have GraphQL error like this, It will send the error to Sentry with traceId so we can trace the error with that info. 
You can see at https://github.com/reapit/foundations/pull/2300/files#diff-bb2ec57b847e57bde9456e46392c444cR30

Reply:
@phmngocnghia I also add traceId to error extensions response.
<img width="470" alt="Screen Shot 2020-08-07 at 11 34 19 AM" src="https://user-images.githubusercontent.com/30467048/89609460-211be800-d8a2-11ea-8097-ce1f7a2ee0a3.png">